README.md

```diff
@@ -19,12 +19,12 @@ Manticore supports binaries of the following formats, operating systems, and
 architectures. It has been primarily used on binaries compiled from C and C++.
 Examples of practical manticore usage are also [on github](https://github.com/trailofbits/manticore-examples).
 
-- OS/Formats: Linux ELF, Windows Minidump
-- Architectures: x86, x86_64, ARMv7 (partial)
+- OS/Formats: Linux ELF
+- Architectures: x86, x86_64, ARMv7
 
 ## Requirements
 
-Manticore is supported on Linux and requires Python 2.7, pip 7.1.0 or higher, and the [Z3 Theorem Prover](https://github.com/Z3Prover/z3/releases). Ubuntu 16.04 is strongly recommended.
```

Comment:
_technically_ manticore doesn't require pip to actually run and do binary analysis. I think it's fair to assume `pip` is available.